google-ads-shaded/pom.xml

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
Please use the license at https://github.com/google/new-project#source-code-headers